api/src/test/scala/hmda/api/http/LarHttpApiSpec.scala

```diff
@@ -51,22 +51,27 @@ class LarHttpApiSpec extends WordSpec with MustMatchers with ScalatestRouteTest
       }
     }
 
-    "filter syntactical or validity only for invalid LAR with both syntactical and validity errors" in {
-      val badLoanType = lar.loan.copy(loanType = 0)
-      val badLar = lar.copy(agencyCode = 0, loan = badLoanType)
-      Post("/lar/validate", badLar) ~> larRoutes ~> check {
-        status mustEqual StatusCodes.OK
-        responseAs[List[ValidationError]].length mustBe 2
-      }
-      Post("/lar/validate?check=syntactical", badLar) ~> larRoutes ~> check {
-        status mustEqual StatusCodes.OK
-        responseAs[List[ValidationError]].length mustBe 1
+    "filter syntactical, validity, or quality only for invalid LAR with " +
+      "both syntactical, validity, and quality errors" in {
```

Comment:
this is a nitpick, but "both" doesn't quite work with more than 2 things. maybe "all 3 kinds of errors"?